### PR TITLE
More API spec fixes

### PR DESCRIPTION
**Pull request by tervay**
god bless [jq](https://jqkungfu.com/)

### CODE REVIEW

**Review by fangeugene — COMMENTED** [review]
I'm not sure if this 100% holds true. Sometimes Cheesy Arena doesn't implement all the breakdowns.

The red/blue requirement should definitely be true though.

**Comment by tervay** [comment]
Oof. Ok. I'll write a script to go through every match since 2015 and see which fields ever get missed.

**Comment by fangeugene** [comment]
Actually you might be able to just use `src/backend/common/helpers/score_breakdown_keys.py`

File: src/backend/web/static/swagger/api_v3.json [hunk 1]
```diff
@@ -5846,6 +5846,12 @@
       "Match_Score_Breakdown_2015": {
         "type": "object",
         "properties": {
+          "required": [
```
Comment (fangeugene) [inline src/backend/web/static/swagger/api_v3.json:5849]: I think this is tabbed wrong. It should be outside of "properties" right?

Comment (tervay) [reply src/backend/web/static/swagger/api_v3.json:5849]: whoops

File: src/backend/web/static/swagger/api_v3.json [hunk 2]
```diff
@@ -7445,6 +7529,11 @@
           },
           "backup": {
             "type": "object",
+            "nullable": true,
```
Comment (fangeugene) [inline src/backend/web/static/swagger/api_v3.json:7532]: is `backup` really both nullable and not required?

Comment (tervay) [reply src/backend/web/static/swagger/api_v3.json:7532]: Yeah.

Undefined in offseason: https://www.thebluealliance.com/api/v3/event/2022bc/alliances
Undefined in season: https://www.thebluealliance.com/api/v3/event/2022mabos/alliances

Nullable in season: https://www.thebluealliance.com/api/v3/event/2018nyut/alliances

Seems like if an event has a backup on any alliance, every other alliance object has a backup field present, which may be null or an actual backup

Comment (fangeugene) [reply src/backend/web/static/swagger/api_v3.json:7532]: amazing

**Pull request merged**